### PR TITLE
OCPBUGS#2006: Added custom tags to example

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Version(s): 4.10+

Issue: https://issues.redhat.com/browse/OCPBUGS-2006

Link to docs preview: https://51418--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-azure.html#machineset-yaml-azure_creating-machineset-azure

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by mgahagan73** [comment]
When I read the description for the tags I don't get the impression that as a customer I can set and modify them as I see fit, it almost reads as something that is required to be there in order for the cluster to operate rather than something optional I can use for my own resource management needs, I think we should make it more clear that customers can add additional tags/values in their machineset, maybe an example such as:

tags:
  - name: company_division
    value: Legal

From reading the doc I just don't get the impression that this is a customer setable/usable field.

**Comment by snarayan-redhat** [comment]
> When I read the description for the tags I don't get the impression that as a customer I can set and modify them as I see fit, it almost reads as something that is required to be there in order for the cluster to operate rather than something optional I can use for my own resource management needs, I think we should make it more clear that customers can add additional tags/values in their machineset, maybe an example such as:
> 
> tags:
> 
> * name: company_division
>   value: Legal
> 
> From reading the doc I just don't get the impression that this is a customer setable/usable field.

I too had a similar opinion while creating this PR. But this is just a clone of a really old PR that went stale https://github.com/openshift/openshift-docs/pull/21475.I did see lgtm from one of the QEs I suppose, hence thought may be this would be it. I definitely agree what you mentioned. Will incorporate those changes.

**Comment by mgahagan73** [comment]
I like this version better!
/lgtm

File: modules/machineset-yaml-azure.adoc [hunk 1]
```diff
@@ -105,14 +105,17 @@ endif::infra[]
           resourceGroup: <infrastructure_id>-rg <1>
           sshPrivateKey: ""
           sshPublicKey: ""
+          tags:
+            - name: <custom_tag_name> <9>
```
Comment (kcarmichael08) [inline modules/machineset-yaml-azure.adoc:109]: Just checking that the `-` character before the `name` field is supposed to be in there (I didn't see it in the bug text)

File: modules/machineset-yaml-azure.adoc [hunk 2]
```diff
@@ -153,11 +156,11 @@ endif::infra[]
 <6> Specify an image that is compatible with your instance type. The Hyper-V generation V2 images created by the installation program have a `-gen2` suffix, while V1 images have the same name without the suffix.
 <7> Specify the region to place machines on.
 <8> Specify the zone within your region to place machines on. Be sure that your region supports the zone that you specify.
+<9> Optional: Specify custom tags in your machine set. Provide tag name in `<custom_tag_name>` field and the corresponding tag value in `<custom_tag_value>` field.
```
Comment (kcarmichael08) [inline modules/machineset-yaml-azure.adoc:159]: suggestion: Add "the" before "tag name": Provide **the** tag name....

**Review by kcarmichael08 — COMMENTED** [review]
Had a question and a suggestion; otherwise, LGTM!

File: modules/machineset-yaml-azure.adoc (hunk 1 above)
Comment (snarayan-redhat) [reply modules/machineset-yaml-azure.adoc:109]: nice observation! yes, it is required.

**Pull request merged**